manifests/dotnet.yml

```diff
@@ -351,6 +351,25 @@ tests/:
       Test_Otel_Span_With_W3c: v2.42.0
     test_otel_tracer.py:
       Test_Otel_Tracer: v2.8.0
+    test_parametric_endpoints.py:
+      Test_Parametric_DDSpan_Add_Link: missing_feature (add_link parametric endpoint is not implemented)
+      # Set error endpoint does not set the error.type tag on spans
+      Test_Parametric_DDSpan_Set_Error: bug (APMAPI-778)
+      Test_Parametric_DDSpan_Set_Resource: missing_feature (set_resource parametric endpoint is not implemented)
+      Test_Parametric_DDTrace_Baggage: missing_feature (baggage is not supported)
+      # Crash endpoint does not crash the application, it only raises an exception in a thread
+      Test_Parametric_DDTrace_Crash: bug (APMAPI-778)
+      Test_Parametric_DDTrace_Current_Span: missing_feature (current_span parametric endpoint is not implemented)
+      # Tests below are failing due to the start_span endpoint consistently returning 500s, the root cause is unknown
+      Test_Parametric_OtelSpan_Events:  bug (APMAPI-778)
```

Comment:
We need to set `DD_TRACE_OTEL_ENABLED=true` in order to run these test. Can you add that to the library env for the Otel tests? If instead we should enable `DD_TRACE_OTEL_ENABLED=true` by default for the .NET parametric test app, then it fails the test `tests/parametric/test_otel_env_vars.py::Test_Otel_Env_Vars::test_otel_sdk_disabled_set`

Reply:
Yup we have the same issue in python. I would like to avoid setting this environment variable for every test test uses the opentelemetry api. It would be nice to find a way to just set in one place but still allow tests override this env. I can make this change.